tools/build_config/build.sh

```diff
@@ -65,18 +65,10 @@ function run_tests() {
 }
 
 function run_analysis() {
-  local output_dir=$1
-
-  if [ -f "$(which cppcheck)" ]; then
-    echo -e "\nRunning analysis step..."
+  local build_dir=$1
 
-    analysis_cmd="cppcheck --enable=all --inconclusive"
-    analysis_cmd+=" --xml --xml-version=2"
-    analysis_cmd+=" -I ${HORACE_ROOT}/_LowLevelCode/cpp"
-    analysis_cmd+=" ${HORACE_ROOT}/_LowLevelCode/"
-    analysis_cmd+=" 2> ${output_dir}/cppcheck.xml"
-    echo_and_run "${analysis_cmd}"
-  fi
+  echo_and_run "cd ${build_dir}"
+  echo_and_run "make analyse"
```

Comment:
There is no Makefile in the source tree. Do we want to introduce make into the build? Can the same be achieved through CMake